cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaBrokerConfigurationDiff.java

```diff
@@ -201,8 +212,8 @@ private Collection<AlterConfigOp> diff(int brokerId, String desired,
         return updatedCE;
     }
 
-    private void updateOrAdd(String propertyName, Map<String, ConfigModel> configModel, Map<String, String> desiredMap, Collection<AlterConfigOp> updatedCE) {
-        if (!isIgnorableProperty(propertyName)) {
+    private void updateOrAdd(String propertyName, Map<String, ConfigModel> configModel, Map<String, String> desiredMap, Collection<AlterConfigOp> updatedCE, boolean isController) {
+        if (!isIgnorableProperty(propertyName, isController)) {
```

Comment:
`isIgnorableProperty` has option `isCombined`. But you pass into it `isController`. So, either you pass a wrong value there or it is badly named in `isIgnorableProperty`.

Reply:
fixed it